cmd/registry/auth/basic/basic.go

```diff
@@ -74,11 +81,22 @@ func (o *loginOptions) RunBasic(ctx context.Context, args []string) error {
 		return fmt.Errorf("unable to create new store: %w", err)
 	}
 
-	if err := basic.Login(ctx, client, credentialStore, reg, user, token); err != nil {
+	if err := basic.Login(ctx, client, credentialStore, reg, o.username, o.password); err != nil {
 		return err
 	}
 	o.Printer.Verbosef("credentials added to credential store")
 	o.Printer.Success.Println("Login succeeded")
 
 	return nil
 }
+
+func (o *loginOptions) ensureCredentials() error {
```

Comment:
Could we support the use case where the username is given on the command line flag and the password from stdin?